### PR TITLE
Laboratory work #3, Bugakova Darya - 20FPL1

### PR DESCRIPTION
**Pull request by daribbb**

### CODE REVIEW

File: lab_3/main.py [hunk 1]
```diff
@@ -20,7 +21,32 @@ def tokenize_by_sentence(text: str) -> tuple:
          (('_', 'h', 'e', '_'), ('_', 'i', 's', '_'), ('_', 'h', 'a', 'p', 'p', 'y', '_'))
          )
     """
-    pass
+    if not isinstance(text, str) or not text:
+        return ()
+
+    umlauts = {"ü": "ue", "ö": "oe", "ä": "ae", "ß": "ss"}
```
Comment (WhiteJaeger) [inline lab_3/main.py:27]: Good.

**Review by WhiteJaeger — APPROVED** [review]
You are done here.

File: lab_3/target_score.txt [hunk 2]
```diff
@@ -1 +1 @@
-6
\ No newline at end of file
+4
```
Comment (WhiteJaeger) [inline lab_3/target_score.txt:1]: Do not forget to change it - I see that you have done the n-grams.

**Review by WhiteJaeger — CHANGES_REQUESTED** [review]

File: lab_3/target_score.txt (hunk 2 above)
Comment (daribbb) [reply lab_3/target_score.txt]: thanks! i really forgot about it, now everything is changed

**Pull request closed without merging**